qute.jdt/com.redhat.qute.jdt/src/main/java/io/quarkus/runtime/util/StringUtil.java

```diff
@@ -0,0 +1,238 @@
+package io.quarkus.runtime.util;

```

Comment:
In IJ, copying this class would be unnecessary, as we already embed a version of quarkus-core. Why doesn't JDT also have a copy of quarkus-core?

Reply:
I'm not sure that in IJ we embed quarkus-core.

The StringUtil exists in IJ because it comes from LSPP4MP which had a copy/paste of this StringUtils in the MP LS.